### ISSUE TITLE
AME Wizard not able to run without internet connection

### ISSUE THREAD

**Issue by mrx-00** [Ameliorated-LLC/trusted-uninstaller-cli#3]
Hi everyone,

During installation of AtlasOS I noticed that it's not possible to run the AME Wizard without internet connection, although I would expect it to be possible at least with limited functionality.

I already posted this by mistake in the AtlasOS repository, kindly refer to [the issue over there](https://github.com/Atlas-OS/Atlas/issues/1195) for details.

Best regards :)

**Comment by he3als** [comment]
For development clarification, it would be nice to have a separate requirement that allows users to skip the internet step (with a conditional tag for actions like `ifInternet: true`) or pre-download the required dependencies beforehand.

**Comment by QueenieArtemisia** [comment]
Yes, this is a must-have feature. Playbooks should be able to be deployed 100% offline. Install kits for the pre-installed programs could be downloaded beforehand and the Wizard could use them. Or just have the Wizard continue without those programs if the internet is not connected.

I have had tremendous difficulties deploying AME to 40+ computers with a faulty internal network. This feature would solve SO MANY issues. If this could be done in a future update, certain people will surely shower you on Patreon :).

Thank you for your hard work.

Cheers
